docs/02-client-tools.md

```diff
@@ -99,6 +99,13 @@ chmod +x kubectl
 ```
 sudo mv kubectl /usr/local/bin/
 ```
+The following steps will enable shell tab completion on kubectl while using bash:
+
+```
+echo "source <(kubectl completion bash)" >> ~/.bashrc && .~/.bashrc
```

Comment:
`.~/.bashrc` is it that correct (no space between dot a ~)? or it should be  `. ~/.bashrc` (with space between dot and ~)

Reply:
totally missed the space